raiden_contracts/tests/fixtures/channel_test_values.py

```diff
@@ -0,0 +1,22 @@
+from collections import namedtuple
+
+
+ChannelValues = namedtuple('ChannelValues', [
+    'deposit',
+    'withdrawn',
+    'transferred',
+    'locked'
+])
+
+channel_test_vals = [
```

Comment:
In the codebase we try not to use abbrevitions, especially if they don't save much space. Can you change `vals` to `values`?

Reply:
Used that name because I called the fixture `channel_test_values`, but as you suggested in another comment, I'll use `pytest.mark.parametrize()` and also change the name.